app/views/trips/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<h2>Trip Details</h2>
+
+
+<p>
+  <strong>
+    <p>Trip date: <%= @trip.date %></p>
+    <% if @trip.rating == nil %>
+    <% @trip.rating = "Trip unfinished" %>
+    <%end%>
+    <p>Trip rating: <%= @trip.rating %></p>
+    <% if @trip.cost == nil %>
+    <% @trip.cost = "Trip unfinished" %>
+    <% else %>
+    <% @trip.cost = ((@trip.cost) / 100) %>
+    <%end%>
+    <p>Trip cost: $<%= @trip.cost %>
+    <p>Driver id: <%= @trip.driver_id %></p>
+    <p>Passenger id: <%= @trip.passenger_id %></p>
```

Comment:
This is not very helpful to only see the passenger's ID number, why not the name?